demo/package.json

```diff
@@ -2,19 +2,7 @@
   "name": "demo",
   "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "classnames": "^2.2.6",
-    "openfin-layouts": "^1.0.3",
-    "openfin-react-hooks": "file:../",
-    "prismjs": "^1.16.0",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-router-dom": "^5.0.0",
-    "react-scripts": "3.0.1",
-    "react-switch": "^5.0.0",
-    "typescript": "3.4.5",
-    "uuid": "^3.3.2"
-  },
+  "dependencies": {},
```

Comment:
Shouldn't we keep actual runtime necessities as dependencies here? Typescript and react-scripts can move over to devDependencies but the rest should stay.

Reply:
That's a good point @oriondean - the code isn't bundled, so those dependencies are required at runtime. I've updated and rebased.